src/averageAges.js

```diff
@@ -54,9 +66,29 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
+function calculateAvg(arr) {
```

Comment:
It is better to not shortcut the words for naming
And think for more descriptive name for arguments (maybe `numbers`, `values`)